aiida/tools/dbimporters/plugins/test_materialsproject.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+###########################################################################
+# Copyright (c), The AiiDA team. All rights reserved.                     #
+# This file is part of the AiiDA code.                                    #
+#                                                                         #
+# The code is hosted on GitHub at https://github.com/aiidateam/aiida_core #
+# For further information on the license, see the LICENSE.txt file        #
+# For further information please visit http://www.aiida.net               #
+###########################################################################
+"""
+Module that contains the class definitions necessary to offer support for
+queries to Materials Project.
+"""
+
+from __future__ import absolute_import
+import pytest
+from aiida.tools.dbimporters import DbImporterFactory
+from aiida.backends.testbase import AiidaTestCase
+
+
+class TestMaterialsProject(AiidaTestCase):
+    """
+    Contains the tests to verify the functionality of the Materials Project importer
+    functions.
+    """
+
+    def test_invalid_api_key(self):  # pylint: disable=no-self-use
```

Comment:
As a minor comment, I would return, in `run_materialsproject_api_tests`:
```python
return profile_conf.get('run_materialsproject_api_tests', False)
```
So that you can still keep the property in there and set it to True/False.
After this, it is good to merge (you might just want to add a comment saying that this can be set via the profile configuration).

Reply:
Okey. Fixed.